core/controllers/question_editor.py

```diff
@@ -25,12 +25,47 @@
 import feconf
 
 
+class QuestionCreationHandler(base.BaseHandler):
+    """A handler that creates the question model given a question dict."""
+
+    @acl_decorators.can_manage_question_skill_status
+    def get(self):
+        """Handles GET requests."""
+        if not feconf.ENABLE_NEW_STRUCTURES:
+            raise self.PageNotFoundException
+
+        new_question_id = question_services.get_new_question_id()
```

Comment:
This also seems wrong. Why get the ID at all? The ID should be assigned for the first time only when the completed question dict reaches the backend after editing and submission (it can be None in the interim).

Reply:
Oh, ok. I thought an ID could be returned from the backend when question is created to be used in the frontend. Yeah, will change this.

Reply:
Done.